sources/bin/shared.py

```diff
@@ -82,3 +82,29 @@ def get_request_json(url: str, headers: dict) -> dict:
 
 def format_date(date: str) -> str:
     return datetime.fromisoformat(date).strftime("%Y-%m-%d")
+
+
+def shorten_tool_id(tool: str) -> str:
+    """
+    Shorten tool id
+    """
+    if "toolshed" in tool:
+        return tool.split("/")[-2]
```

Comment:
are you sure it's not `[-1]` ?

Reply:
If we want to have the suite id it is `[-2]` and it is what we used there: https://github.com/galaxyproject/galaxy_codex/pull/158/files#diff-fded0df90a63fa85d73a79a61797665ed61e1b1a921558f0d3b301d877a54dc4L38